### PR TITLE
Updating RPI configuration for harware float and kernel version

### PR DESCRIPTION
**Pull request by stef3569**
Because armv6-rpi-linux-gnueabi is outdated (hardware FPU and kernel version)
Exact copy of current armv6-rpi-linux-gnueabi configuration except for hard float and kernel version.

---
This change is [Reviewable](https://reviewable.io/reviews/crosstool-ng/crosstool-ng/377)

### CODE REVIEW

**Comment by yann-morin-1998** [comment]
Stéphane,

You forgot to sign-off your commits.

Besides, I'll add some comments in the patches themselves.

File: samples/armv6-rpi-linux-gnueabihf/crosstool.config [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+CT_TARGET_VENDOR="rpi"
+CT_KERNEL_linux=y
+CT_KERNEL_V_4_1=y
+CT_BINUTILS_LINKER_LD_GOLD=y
+CT_BINUTILS_GOLD_THREADS=y
+CT_BINUTILS_LD_WRAPPER=y
+CT_BINUTILS_PLUGINS=y
+CT_LIBC_LOCALES=y
+CT_LIBC_GLIBC_KERNEL_VERSION_CHOSEN=y
+CT_LIBC_GLIBC_MIN_KERNEL_VERSION="3.2.27"
```
Comment (yann-morin-1998) [inline samples/armv6-rpi-linux-gnueabihf/crosstool.config]: Stéphane,

Explain why you kept glibc support for that kernel:  that's the oldest kernel
with support for the RPi-1, so no need to have backward compatibility with
older kernels.

File: samples/armv6-rpi-linux-gnueabihf/reported.by [hunk 2]
```diff
@@ -0,0 +1,3 @@
+reporter_name="Yann E. MORIN"
```
Comment (yann-morin-1998) [inline samples/armv6-rpi-linux-gnueabihf/reported.by]: Stéphane,

I guess you should have changed that to your own identity! ;-)
(ditto for the URL, below).

**Comment by bhundven** [comment]
As per: https://sourceware.org/ml/crossgcc/2016-03/msg00023.html
I would like to see the sample names change to be generic. There are other suitable target boards that fit these configurations. Hence these configuration names shouldn't be specific to RPI.

**Comment by bhundven** [comment]
In other words, rather then removing armv6-rpi-linux-gnueabi and adding armv6-rpi-linux-gnueabihf, I would like to see:

armv6-rpi-linux-gnueabi -> armv6-unknown-linux-gnueabi
armv6-rpi-linux-gnueabihf -> armv6-unknown-linux-gnueabihf

**Comment by bhundven** [comment]
@stef3569 Could you rename the samples as I mentioned? Then I will merge this PR.

**Pull request closed without merging**